papers/2024_Abderezaei_IROS_SFRRT.bib

```diff
@@ -0,0 +1,6 @@
+@inproceedings{abderezaei2024clutterawarespillfreeliquidtransport,
+  title={Clutter-Aware Spill-Free Liquid Transport via Learned Dynamics}, 
+  author={Abderezaei, Ava and Pasricha, Anuj and Klausenstock, Alex and Roncone, Alessandro},
+  year={2024},
+  booktitle={2024 IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS)},
+}
```

Comment:
add a new line at the end of file in this file please

Reply:
Done!